### DISCUSSION TITLE
Fine tuning via onnxruntime-genai C#

### DISCUSSION THREAD

**Discussion by svsnux** [microsoft/onnxruntime-genai#703] [New features / APIs]
Fine tuning of a Model like Phi-3 via OnnxRuntime-genai C# would be a very nice to have feature.

**Comment by natke** [comment]
Hi @larcai, can we clarify that you mean having the fine-tuning API in C# rather than Python?

**Comment by kunal-vaishnavi** [comment]
You can use [Olive](https://github.com/microsoft/olive) for fine-tuning and then run your fine-tuned model with ONNX Runtime GenAI.
